src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -195,15 +195,18 @@ def run_with_retry(retries, interval)
 # Poll the status of a Kubernetes namespace, until all the pods in that
 # namespace are ready and all the jobs have run.
 def wait_for_namespace(namespace, sleep_duration=10)
+    STDERR.puts "#{c_bold}# Waiting for: #{namespace}, in increments of #{sleep_duration}s#{c_reset}"
```

Comment:
That seems a bit too verbose.

Reply:
Will compress. (Tomorrow).

Reply:
Done with commit e8a0eca19f6e6b69f9f685bda659c9a87380d02c.